src/features/machines/getBalanceMachine.ts

```diff
@@ -133,3 +134,19 @@ export const getNearNep141StorageBalance = async ({
     throw new Error("Error fetching balance", { cause: err })
   }
 }
+
+export const getNearNep141MinStorageBalance = async ({
+  contractId,
+}: {
+  contractId: string
+}): Promise<bigint> => {
+  const response = await getNearNep141StorageBalanceBounds({
+    request_type: "call_function",
+    method_name: "storage_balance_bounds",
+    account_id: contractId,
+    args_base64: "e30=",
+    finality: "optimistic",
+  })
+  const balance = JSON.parse(Buffer.from(response.result).toString())
```

Comment:
I would avoid using Buffer, because browsers don't have it. We can write it like this:
```js
  const uint8Array = new Uint8Array(response.result)
  const decoder = new TextDecoder()
  const parsed = JSON.parse(decoder.decode(uint8Array))
  ```